### DISCUSSION TITLE
Missing some buttons on a smaller window

### DISCUSSION THREAD

**Discussion by swith-wiz-kev** [community/community#37650] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/33258487/198684007-219cb74d-4e4e-4447-981e-4892efdfc469.png)

The fork button is missing, there is no way to fork unless you have a wide screen. This might be issue on vertical monitors too.

**Comment by jfullstackdev** [comment]
that's being responsive site, actually from the past, they have improve a lot,

on vertical monitors, I'm not very sure, but if that is standard size, maybe it will not be missing those buttons

this is from a 21" Portrait 1080p

![image](https://user-images.githubusercontent.com/47092464/198884223-7c8f586b-3cf9-48b0-b3f3-d5004b617322.png)

and 24" Portrait 1080p

![image](https://user-images.githubusercontent.com/47092464/198884313-5173cbec-8f0d-43f8-b98f-ab5a559d3045.png)
